aws_helper/config.go

```diff
@@ -188,6 +235,35 @@ func AssumeIamRole(iamRoleOpts options.IAMRoleOptions) (*sts.Credentials, error)
 		sessionDurationSeconds = iamRoleOpts.AssumeRoleDuration
 	}
 
+	if iamRoleOpts.WebIdentityToken != "" {
```

Comment:
Nested block can be simplified
```
if iamRoleOpts.WebIdentityToken == "" {
// return 
}
// continue flow when WebIdentityToken is set

Reply:
It isn't as simple as putting a `return` here, but I flipped the logic flow so AssumeRoleWithWebIdentity is the default case.